site/src/pages/WorkspacePage/WorkspaceActions/DownloadLogsDialog.tsx

```diff
@@ -163,4 +275,22 @@ const styles = {
   listItemSecondary: {
     fontSize: 14,
   },
+
+  notAvailableText: (theme) => ({
+    display: "flex",
+    flexFlow: "row nowrap",
+    alignItems: "center",
+    columnGap: "4px",
+
+    "& > span": {
+      maxHeight: "fit-content",
+      display: "flex",
+      alignItems: "center",
+      color: theme.palette.error.light,
+    },
+
+    "& > p": {
+      opacity: "80%",
+    },
+  }),
```

Comment:
I've been stuck in the pre-Emotion, Backstage-specific version of MUI for the past few months, so please let me know if anything looks funky